server/etcdmain/grpc_proxy.go

```diff
@@ -430,9 +435,32 @@ func newGRPCProxyServer(lg *zap.Logger, client *clientv3.Client) *grpc.Server {
 	electionp := grpcproxy.NewElectionProxy(client)
 	lockp := grpcproxy.NewLockProxy(client)
 
+	alwaysLoggingDeciderServer := func(ctx context.Context, fullMethodName string, servingObject interface{}) bool { return true }
+
+	grpcChainStreamList := []grpc.StreamServerInterceptor{
+		grpc_prometheus.StreamServerInterceptor,
+	}
+	grpcChainUnaryList := []grpc.UnaryServerInterceptor{
+		grpc_prometheus.UnaryServerInterceptor,
+	}
+	if grpcProxyEnableLogging {
+		grpcChainStreamList = append(grpcChainStreamList,
+			grpc_ctxtags.StreamServerInterceptor(),
+			grpc_zap.PayloadStreamServerInterceptor(lg, alwaysLoggingDeciderServer),
+		)
+		grpcChainUnaryList = append(grpcChainUnaryList,
+			grpc_ctxtags.UnaryServerInterceptor(),
```

Comment:
There is no any option being provided to function [grpc_ctxtags.UnaryServerInterceptor](https://github.com/grpc-ecosystem/go-grpc-middleware/blob/854bd94ac7c4b4d287b9eba9b7cc4410e7215fed/tags/interceptors.go#L16), so the [o.requestFieldsFunc](https://github.com/grpc-ecosystem/go-grpc-middleware/blob/854bd94ac7c4b4d287b9eba9b7cc4410e7215fed/tags/interceptors.go#L20) will be always `nil`. Accordingly it will do nothing. 

Did I miss anything?

Reply:
https://github.com/grpc-ecosystem/go-grpc-middleware/blob/854bd94ac7c4b4d287b9eba9b7cc4410e7215fed/tags/interceptors.go#L70

has a default `peer.address` tags